### ISSUE TITLE
Picker search bar needs aria-label

### ISSUE THREAD

**Issue by cheygo** [ahkohd/ngx-emoji-picker#47]
While running accessibility checks for my app I am noticing failures on the search bar within the emoji-picker. It just needs a simple aria-label for the screenreader to make this error go away. Here is a picture of the error: 
![Screen Shot 2020-07-15 at 1 24 24 PM](https://user-images.githubusercontent.com/69327675/89589133-051d3580-d813-11ea-9c8a-b92bb3f15a24.png)

**Comment by ahkohd** [comment]
Hi, @cheygo thanks for reporting this.
I apologize for the late response. I'll look into it and push an update a soon as possible.

**Comment by ahkohd** [comment]
@cheygo #47 commit contains fix to this issue. I'll let you know once I push the updated package to **NPM**.

**Comment by ahkohd** [comment]
@all-contributors please add @cheygo  for reporting ally.

**Comment by ahkohd** [comment]
@all-contributors please add @cheygo  for reporting a11y.

**Comment by ahkohd** [comment]
🎉 Hi @cheygo I have published **ngx-emoji-picker@0.1.6** it contains the fix the reported accessibility issue and much more: #50  
```bash
npm i ngx-emoji-picker twemoji
```
